css/style.css

```diff
@@ -13,12 +13,13 @@ html, button, input, select, textarea,
 
 div.rustdoc {
   font-family: "Source Serif Pro", Georgia, Times, "Times New Roman", serif;
-  padding: 10px 15px 20px 15px;
```

Comment:
It looks like taking out this 'padding' style makes the search bar overlap with the style button, at least in my browser.

![image](https://user-images.githubusercontent.com/23638587/68075822-9abafe80-fd83-11e9-81e9-a43e44bd8c27.png)

Reply:
Oh funny. Can you open an issue about this (and maybe even a PR?). I don't want to mix more things in this PR.

Reply:
I'm not sure I understand, do you mean in the main rust repo? If you just add the padding back in (like it was before) it works fine.

Reply:
Oh crap! Completely missed the fact that I removed it. Good catch! Sorry for not understanding at first.